apps/api/src/common/slack.ts

```diff
@@ -0,0 +1,35 @@
+import { Logger } from '@nestjs/common';
+import axios from 'axios';
+import yaml from 'js-yaml';
+import https from 'https';
+
+const webhookUrl = process.env.SLACK_ALERTS_WEBHOOK_URL;
+
+export default function postToSlack(data: unknown): void {
```

Comment:
we should handle slack push failure gracefully. This is not main ops.... so in any case it should not tigger any failure. Mean, put whole code into a try/catch  otherwise that can create recursive execution.